challenges/van-renting.md

```diff
@@ -0,0 +1,41 @@
+#Article application for booking:
+
+So, an article shows information about a location and allows a user to pick checkin and checkout dates and book directly on that page. 
+
+In order to create a booking you need the checkin and checkout cities, checkin and checkout dates, the van and the client name, email and nationality. 
+
+An article must show a title and a description (the article body) and a photo of the location. After the client books for a location, he should receive an email confirming the booking (you don't need to be fancy. 
+
+A simple string will do. Also if you want to, you can also localize this email). To simplify the information, create only 3 cities (Lisbon, Porto and Faro), one article per city, one van model and 3 vans of that model. 
+
+You don’t need to create an interface to manage these objects but keep in mind that they can change and/or be deleted at any time. 
+
+
+# Context
+
+The goal of this challenge is to develop an article application that allows a user to list articles and see the article in order to book a van to travel to the location/city shown in that article
+
+# Goal
+
+An article shows information about a location and allows a user to pick checkin and checkout dates and book directly on that page. 
+
+An article must show a title and a description (the article body) and a photo of the location. After the client books for a location, he should receive an email confirming the booking (a simple string will do). 
+
+To simplify the information, create only 3 cities (Lisbon, London and Paris), one article per city, one van model and 3 vans of that model.
+
+You don’t need to create an interface to manage these objects but keep in mind that they can change and/or be deleted at any time.
+
+UI/UX is up to you (no need for fanciness, simple _divs_ will do)
+
+# Constraints
+
+- Must me multi-language (both static and "database" content)
+
+## Tooling
+
+- Rails
+- Asynchronous Email
+
+## Timeline
+
+2 weeks to deliver full product
```

Comment:
"2 weeks to deliver full product" => "You have 2 weeks to deliver the completed application."